src/v2/WidgetRouter.js

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/* eslint max-params: [2, 50] */
+define([
+  'okta',
+  './util/BaseLoginRouter',
+  './controllers/RefreshAuthStateController',
+  './controllers/FormController',
+  '../views/shared/SecurityBeacon',
+  '../views/shared/FactorBeacon',
+],
+function (Okta,
+  BaseLoginRouter,
+  RefreshAuthStateController,
+  FormController,
+  SecurityBeacon,
+  FactorBeacon) {
+  return BaseLoginRouter.extend({
+    routes: {
+      '': 'defaultAuth',
+      'signin/refresh-auth-state(/:token)': 'refreshAuthState',
+      'signin/render': 'renderWidgetView',
```

Comment:
this is doing same job with default Auth. sounds redundant ?

Reply:
I can remove this for now since we havent built UNAUTHENTICATED state. But ideally defaultAuth should load primaryAuth, i.e it will call renderWidgetView but some how force it to load login screen

Reply:
> defaultAuth should load primaryAuth,
then might be better to just call it primaryAuth.

In V2 world, I thought the View shall be always determined by Token and there are not any stateless status?

Reply:
Yes, we should always have a statetoken in v2 world. We could use `defaultAuth`/ `primaryAuth` to show login screen when we hit the widget with an invalid/expired token or as a fallback route ?. Currently it shows this screen.


<img width="442" alt="Screen Shot 2019-07-09 at 5 42 56 PM" src="https://user-images.githubusercontent.com/23267876/60932063-2529a080-a271-11e9-9962-8ffcf8fde0f2.png">

Reply:
interesting. sounds like we could fall back to primary auth for now?